app/src/main/java/org/groundplatform/android/ui/common/BindingAdapters.kt

```diff
@@ -52,14 +52,11 @@ object BindingAdapters {
     if (view == null || uri == null) return
 
     try {
-      Picasso.get()
-        .load(uri)
-        .placeholder(R.drawable.ic_photo_grey_600_24dp)
-        .transform(RotateUsingExif(uri, view.context))
-        .into(view)
-    } catch (exception: IllegalStateException) {
-      // Needed for running unit tests
-      Timber.e(exception)
+      val rotatedBitmap = loadBitmapWithCorrectOrientation(view.context, uri)
+      view.setImageBitmap(rotatedBitmap)
+    } catch (e: Exception) {
```

Comment:
It's generally not advisable to use a generic "catch-all" for exceptions. A more robust approach is to explicitly list and handle the specific exceptions you expect to occur. In cases where no particular exceptions are foreseen, it's often better to allow the program to fail gracefully rather than attempting to catch an unknown error.